### PR TITLE
Updated statuses.py

### PR DESCRIPTION
**Pull request by Sophyia7**
I added new quotes to the statuses.py file. 

Twitter handle: @sophiairoegbu_

### CODE REVIEW

File: statuses.py [hunk 1]
```diff
@@ -73,5 +73,9 @@
    "I love hugging. I wish I was an octopus so I could hug ten people at a time.",
    "Hug me and you will see the moon. Love me and I will bring the moon to you!",
    "Hugs and kisses are ways to express what cannot be said.",
-   "Sometimes all you can do is hug a friend tightly and wish that their pain could be transferred by touch to your own emotional hard drive."
+   "Sometimes all you can do is hug a friend tightly and wish that their pain could be transferred by touch to your own emotional hard drive.",
+   "In your darkest, hug someone. It helps!",
+   "Live, Love, Hug",
```
Comment (achasveachas) [inline statuses.py:78]: Please add a period at the end.

File: statuses.py [hunk 2]
```diff
@@ -73,5 +73,9 @@
...
+   "Live is simply better when you receive and give a hug",
```
Comment (achasveachas) [inline statuses.py:79]: Looks like there's a typo and the first word should be Life"?

Also, a period at the end :)

File: statuses.py [hunk 3]
```diff
@@ -73,5 +73,9 @@
...
+   "Go out and hug someone today!"
```
Comment (achasveachas) [inline statuses.py:80]: The convention I was following was having a trailing comma after the last sentence in the list (it makes for cleaner diffs in subsequent commits/PRs).

(I realize the last person forgot to add it, and i guess it's on me for not catching it, but lets add it here for the future).

**Review by achasveachas — CHANGES_REQUESTED** [review]
Thanks! These are awesome! 🤗 

A few minor fixes and then I will merge it.

File: statuses.py (hunk 3 above)
Comment (Sophyia7) [reply statuses.py]: Thanks for catching these errors early. I have made the changes, please check them out.

Comment (achasveachas) [reply statuses.py]: Thanks! Looks good! 🤗

**Review by achasveachas — APPROVED** [review]

**Pull request merged**